### PR TITLE
MORE LOADOUT STUFF

### PR DESCRIPTION
**Pull request by ghost**
[why]: # muh customization
also i did all this on the browser so i never tested if i accidentally broke everything, will properly do stuff later when i get home
:cl:
add: Added more loadout options
add: Added a cosmetic version of the goliath cloak
/:cl:

### CODE REVIEW

**Comment by ghost** [comment]
don't approve it if people keep suggesting more stuff, or do whatever

**Review by deathride58 — CHANGES_REQUESTED** [review]
Modularize and remove armor values from the cloak

**Comment by ghost** [comment]
yeah i removed the armor values

**Comment by ghost** [comment]
right? Since it's cloak/david and not cloak/goliath/david

File: code/modules/clothing/suits/cloaks.dm [hunk 1]
```diff
@@ -71,6 +71,19 @@
 	desc = "A protective & concealing hood."
 	armor = list("melee" = 35, "bullet" = 10, "laser" = 25, "energy" = 10, "bomb" = 25, "bio" = 0, "rad" = 0, "fire" = 60, "acid" = 60)
 	flags_inv = HIDEEARS|HIDEEYES|HIDEHAIR|HIDEFACIALHAIR
+	
+/obj/item/clothing/suit/hooded/cloak/david
```
Comment (deathride58) [inline code/modules/clothing/suits/cloaks.dm]: Move these changes to a file in modular_citadel

**Review by deathride58 — CHANGES_REQUESTED** [review]

**Comment by ghost** [comment]
k let me just add some other loadout options

**Comment by ghost** [comment]
k should be good

**Review by deathride58 — DISMISSED** [review]

File: modular_citadel/code/modules/client/loadout/uniform.dm [hunk 2]
```diff
@@ -123,6 +141,40 @@
...
+	name = "Pink stripper outfit"
+	category = slot_w_uniform
+	path = /obj/item/clothing/under/stripper_pink
+	cost = 8 //thots pay the thot toll
+
+/datum/gear/strpgreen
+	name = "Green stripper outfit"
+	category = slot_w_uniform
+	path = /obj/item/clothing/under/stripper_green
+	cost = 8
```
Comment (silicons) [inline modular_citadel/code/modules/client/loadout/uniform.dm]: no

File: modular_citadel/code/modules/client/loadout/uniform.dm [hunk 3]
```diff
@@ -123,6 +141,40 @@
 	category = slot_w_uniform
 	path = /obj/item/clothing/under/syndicate/cosmetic
 
+/datum/gear/strppink
+	name = "Pink stripper outfit"
+	category = slot_w_uniform
+	path = /obj/item/clothing/under/stripper_pink
+	cost = 8 //thots pay the thot toll
```
Comment (silicons) [inline modular_citadel/code/modules/client/loadout/uniform.dm]: no

File: modular_citadel/code/modules/client/loadout/gloves.dm [hunk 4]
```diff
@@ -1,4 +1,28 @@
...
 	name = "Fingerless Gloves"
 	category = slot_gloves
-	path = /obj/item/clothing/gloves/fingerless
\ No newline at end of file
+	path = /obj/item/clothing/gloves/fingerless
+
+/datum/gear/glovesrainbow
+	name = "Rainbow gloves"
+	category = slot_gloves
+	path = /obj/item/clothing/gloves/color/rainbow
```
Comment (improvedname) [inline modular_citadel/code/modules/client/loadout/gloves.dm]: Hides fingerprints

File: modular_citadel/code/modules/client/loadout/gloves.dm [hunk 5]
```diff
@@ -1,4 +1,28 @@
...
+	cost = 2
+
+/datum/gear/glovesred
+	name = "Red gloves"
+	category = slot_gloves
+	path = /obj/item/clothing/gloves/color/red
```
Comment (improvedname) [inline modular_citadel/code/modules/client/loadout/gloves.dm]: Hides fingerprints

File: modular_citadel/code/modules/client/loadout/gloves.dm [hunk 6]
```diff
@@ -1,4 +1,28 @@
...
+	cost = 2
+
+/datum/gear/glovesblue
+	name = "Blue gloves"
+	category = slot_gloves
+	path = /obj/item/clothing/gloves/color/blue
```
Comment (improvedname) [inline modular_citadel/code/modules/client/loadout/gloves.dm]: Hides fingerprints

File: modular_citadel/code/modules/client/loadout/gloves.dm [hunk 7]
```diff
@@ -1,4 +1,28 @@
...
+	cost = 2
+
+/datum/gear/glovesgray
+	name = "Gray gloves"
+	category = slot_gloves
+	path = /obj/item/clothing/gloves/color/grey
```
Comment (improvedname) [inline modular_citadel/code/modules/client/loadout/gloves.dm]: Hides fingerprints

**Comment by improvedname** [comment]
increasing the cost won't do
because the meta would be everyone wear gloves and it makes detectives job pain in the ass do.

just remove them tbh ill do it later

**Comment by improvedname** [comment]
🖒

**Comment by ghost** [comment]
>conflicts

UGHHH

**Comment by ghost** [comment]
this would be easier if i was at home and had smartgit

**Comment by ghost** [comment]
ahh how do i edit uniform without conflicts

**Comment by deathride58** [comment]
@Commandersand Ready for merge?

**Comment by ghost** [comment]
uhh god yeah sure it's fien

**Review by deathride58 — APPROVED** [review]

**Pull request merged**